File: Exercicio-13.java
import java.util.Scanner;
import java.text.DecimalFormat; 

public class Main
{
  public static void main (String[]args)
  {
    Scanner sc = new Scanner (System.in);
    DecimalFormat df = new DecimalFormat("#,###.0");
    double weight, weightGrams, newWeightGrams;

    System.out.printf("Digite o seu peso: ");
    weight = sc.nextDouble();
    
    weightGrams = weight * 1000;
    newWeightGrams = (weight + (weight * 5 / 100)) * 1000;
  
    System.out.println("Peso: " + df.format(weight));
    System.out.println("Peso em Gramas: " + df.format(weightGrams));
    System.out.println("Novo peso em Gramas: " + df.format(newWeightGrams));
  }
}

File: Exercicio-13.c
#include <stdio.h>

int main(void) {

  float weight, weightGrams, newWeightGrams;

  printf("Digite o seu peso: ");
  scanf("%f", &weight);
  
  weightGrams = weight * 1000;
  newWeightGrams = (weight + (weight * 5 / 100)) * 1000;
  
  printf("\nPeso: %.1f \n", weight);
  printf("Peso em Gramas: %.1f \n", weightGrams);
  printf("Novo peso em Gramas: %.1f \n", newWeightGrams);
  
  return 0;
}
